src/ui/flutter_app/lib/screens/game_page/game_page.dart

```diff
@@ -68,129 +65,91 @@ class _GamePageState extends State<GamePage>
+    // sqrt(1.618) = 1.272
+    _animation = Tween(begin: 1.27, end: 1.0).animate(_animationController)
       ..addListener(() => setState(() {}));
 
     _animationController.addStatusListener((state) {
       if (state == AnimationStatus.completed ||
           state == AnimationStatus.dismissed) {
-        if (disposed) {
-          return;
-        }
-        _animationController.forward();
+        if (!_disposed) _animationController.forward();
       }
     });
-
-    if (!disposed) {
-      _animationController.forward();
-    }
   }
 
-  void showTip(String tip) {
+  void _showTip(String tip, {bool snackBar = false}) {
     if (!mounted) return;
 
     debugPrint("[tip] $tip");
-    if (LocalDatabaseService.preferences.screenReaderSupport) {
-      //showSnackBar(context, tip);
+    if (LocalDatabaseService.preferences.screenReaderSupport && snackBar) {
+      ScaffoldMessenger.of(context).clearSnackBars();
+      showSnackBar(context, tip);
     }
 
     setState(() => _tip = tip);
   }
 
-  void showTips() {
+  void _showTips() {
     if (!mounted) return;
 
     final winner = gameInstance.position.winner;
 
     switch (winner) {
       case PieceColor.white:
-        return showTip(S.of(context).whiteWin);
+        return _showTip(S.of(context).whiteWin);
       case PieceColor.black:
-        return showTip(S.of(context).blackWin);
+        return _showTip(S.of(context).blackWin);
       case PieceColor.draw:
-        return showTip(S.of(context).isDraw);
+        return _showTip(S.of(context).isDraw);
       case PieceColor.nobody:
         switch (gameInstance.position.phase) {
           case Phase.placing:
-            return showTip(S.of(context).tipPlace);
+            return _showTip(S.of(context).tipPlace);
           case Phase.moving:
-            return showTip(S.of(context).tipMove);
+            return _showTip(S.of(context).tipMove);
           default:
         }
     }
 
     if (!LocalDatabaseService.preferences.isAutoRestart) {
-      showGameResult(winner);
+      _showGameResult(winner);
     }
   }
 
-  Future<dynamic> onBoardTap(int index) async {
-    if (!isReady) {
-      debugPrint("[tap] Not ready, ignore tapping.");
-      return false;
-    }
+  Future<void> _onBoardTap(int index) async {
+    if (!mounted) return debugPrint("[tap] Not ready, ignore tapping.");
 
-    disposed = false;
-    _animationController.duration = Duration(
-      milliseconds:
-          (LocalDatabaseService.display.animationDuration * 1000).toInt(),
-    );
+    _disposed = false;
     _animationController.reset();
 
-    if (gameInstance.engineType == EngineType.aiVsAi ||
-        gameInstance.engineType == EngineType.testViaLAN) {
-      debugPrint("$_tag Engine type is no human, ignore tapping.");
-      return false;
+    if (widget.engineType == EngineType.aiVsAi ||
+        widget.engineType == EngineType.testViaLAN) {
+      return debugPrint("$_tag Engine type is no human, ignore tapping.");
```

Comment:
Strange, why do you return this?

Reply:
this returns void so nothing. 
I wrote it as it isn't two lines anymore (in this file this is really important to get an overview xD)